packages/report-server/examples.http

```diff
@@ -39,25 +39,22 @@ Authorization: {{authorization}}
     "transform": [
       "keyValueByDataElementName",
       {
-          "$title": "Calculate occuptions from data elements",
-          "$description": "Sums the raw data elements values to their appropriate occuption",
-          "transform": "select",
-          "'Doctors'": "$row.BCD46",
-          "'Midwives'": "$row.BCD47",
-          "'Nurses'": "sum($row.BCD48, $row.BCD48a, $row.BCD48b)",
-          "'Aides'": "$row.BCD49",
-          "'Others'": "sum($row.BCD50, $row.BCD51, $row.BCD52, $row.BCD53, $row.BCD54, $row.BCD55)",
-          "...": ["period", "organisationUnit"]
+          "title": "Calculate occuptions from data elements",
+          "description": "Sums the raw data elements values to their appropriate occuption",
+          "transform": "updateColumns",
+          "insert": {
+            "Doctors": "$row.BCD46",
```

Comment:
Note: Switching the `$row.x` to `$x` change is a bit trickier than it first seems. Since mathjs throws an error for any undefined symbol, we actually need to do some pre-processing on symbols with `$` at the start to ensure we can handle the `undefined` case. Mathjs fully supports this, and even have documented how to do this: https://mathjs.org/examples/advanced/custom_scope_objects.js

However, I feel it's a bit out of the scope of these refactors, as the correct place to do this would be in the `expression-parser` package itself. So for now we will still live with `$row.x` for a bit longer?

Reply:
Ignore this, it turned out to not be too hard to support this so I've added support!